Management API Rate Limits for Self Service Subscribers (#8982)
auth0/docs: articles/policies/entity-limits.md

---
title: Entity Limit Policy
description: Describes Auth0's tenant entity limit policy for subscribers.
topics:
    - auth0-policies
    - rate-limits
    - entity-limits
contentType:
  - reference
useCase:
  - support
---
# Entity Limit Policy

Entities in Auth0 are tenant configuration elements such as applications, connections, rules, and API resource servers. 

> Auth0 limits the number of entities you can have depending on your subscription level. Auth0 provides notifications to you when you are approaching (90%) and when you have reached (100% or higher) your respective entity limits. We will also provide informative messages to prevent you from attempting to configure entities that would end up being rejected.

Reviewer: ```suggestion
Auth0 limits the number of entities you can have depending on your subscription level. Auth0 provides notifications to you when you are approaching (80%) and when you have reached (100% or higher) your respective entity limits. We will also provide informative messages to prevent you from attempting to configure entities that would end up being rejected.
```

Author: manually updated with a separate commit

Here is an example of a message you would see if you reached your connection limit:

![Entity Limit Reached](/media/articles/policies/entity-limit-reached.png)

> ## Enterpise, Developer and Developer Pro subscription limits

Reviewer: We might want to mention that entity limits counts might take some seconds to be updated. If customer is still seeing a warning when they believe they shouldn't, they should try again after some seconds or contact support if the issue persists.

Author: Added a note

> ## Enterpise, Developer and Developer Pro subscription limits

Reviewer 2: ```suggestion
## Developer and Developer Pro subscription limits
```

Please make it clear that Enterprise subscriptions do not have limits.

---
title: Entity Limit Policy
description: Describes Auth0's tenant entity limit policy for subscribers.
topics:
    - auth0-policies
    - rate-limits
    - entity-limits
contentType:
  - reference
useCase:
  - support
---
# Entity Limit Policy

> **This policy is effective for all paid and free subscriptions made before May 19, 2020.** 

Reviewer: ```suggestion
::: note
This policy is effective for all Developer, Developer Pro, and free subscriptions made on or after May 19, 2020. Starting on **June 18, 2020**, the policy will apply to all Developer, Developer Pro, and free subscriptions.
:::
```
